resolwe/test/testcases/process.py

```diff
@@ -535,6 +538,27 @@ def assertStatus(self, obj, status):  # pylint: disable=invalid-name
             msg="Data status is '{}', not '{}'".format(obj.status, status) + self._debug_info(obj)
         )
 
+    def _get_output_field(self, obj, path):
+        """Return object's output field value and schema.
+
+        :param obj: object with the output field
+        :type obj: ~resolwe.flow.models.Data
+
+        :param str path: path to
+            :class:`~resolwe.flow.models.Data` object's output field
+
+        """
+        field_schema, field = None, None
+        for field_schema, field, field_path in iterate_fields(obj.output, obj.process.output_schema, ''):
+            if path == field_path:
+                break
+        else:
+            self.fail("Field not found in path {}.".format(path))
+
+        field_name = field_schema['name']
+        field_value = field[field_name]
+        return field_value, field_schema
```

Comment:
Changed to `return field[field_name], field_schema` so that we can still check field type in [assertDirExists](https://github.com/genialis/resolwe/pull/482/files#diff-9bac5f597553267222418cbf41d04daeR762).